cps/tasks/metadata_extract.py

```diff
@@ -67,9 +67,12 @@ def _fetch_requested_urls(self, conn):
                      if "error" in self.columns
                      else "SELECT path, duration FROM media WHERE path LIKE 'http%'")
             rows = conn.execute(query).fetchall()
-            requested_urls = {row[0]: {"duration": row[1]} if row[1] is not None and row[1] > 0 else self.unavailable.append(row[0]) for row in rows}
-            for url in self.unavailable:
-                requested_urls.pop(url)
+            requested_urls = {}
+            for path, duration in rows:
+                if duration is not None and duration > 0:
+                    requested_urls[path] = {"duration": duration};
```

Comment:
> requested_urls[path] = {"duration": duration};

@deldesir @codewiz is the trailing semicolon (above) truly necessary?

Reply:
No, that was my mistake... I guess for doing too much C++

Reply:
No. I just removed it.

Reply:
Thanks:

https://github.com/iiab/calibre-web/commit/b0adaa0b26e984d3048a4183cf280dcb4a99d3e9